fluid/resnet50.py

```diff
@@ -198,7 +198,7 @@ def run_benchmark(model, args):
             buf_size=5120),
         batch_size=args.batch_size)
 
-    place = core.CPUPlace() if args.device == 'CPU' else core.CUDAPlace(0)
+    place = core.CPUPlace() if args.device == 'CPU' else core.GPUPlace(0)
```

Comment:
I remember CUDAPlace is used in the latest PaddlePaddle. Could you refer to the source codes to confirm this.

Reply:
Done